include/dwfl_module.hpp

```diff
@@ -10,10 +10,21 @@ extern "C" {
 namespace ddprof {
 
 struct DDProfMod {
-  DDProfMod() : _mod(nullptr), _low_addr(0), _high_addr(0) {}
+  enum Status {
+    kUnknown,
+    kInconsistent,
+  };
+
+  DDProfMod() : _mod(nullptr), _low_addr(0), _high_addr(0), _status(kUnknown) {}
+
+  explicit DDProfMod(Status inconsistent) : DDProfMod() {
```

Comment:
Very slight nit.  Even if this method is only called to mark an object as inconsistent in code, there's no such limitation in the implementation itself.  Please feel free to ignore, but proposing
1.  Change the parameter local name to "value" or "status" or anything
2. Change the name of the function to reflect the intent that it only be used to mark objects as inconsistent (and possibly adjust the method logic accordingly)

Again, your discretion--just seemed jarring on first read.

Reply:
Local name should have been renamed, thank you